### ISSUE TITLE
`proc': tried to create Proc object without a block (ArgumentError)

### ISSUE THREAD

**Issue by jasonlor** [whitesmith/rubycritic#389]
Hi there,

I'm encountering an issue with running rubycritic. I'm using ruby 3.0.0 and am seeing this error during the 

```
bundle exec rubycritic testapp/

running flay smells
...................
running flog smells
......................................................................................................................................................................................
running reek smells
......................................................................................................................................................................................
running complexity
......................................................................................................................................................................................
running attributes
......................................................................................................................................................................................
running churn
......................................................................................................................................................................................
bundler: failed to load command: rubycritic (/testapp/vendor/cache/ruby/3.0.0/bin/rubycritic)
/testapp/vendor/cache/ruby/3.0.0/gems/rubycritic-4.6.0/lib/rubycritic/analysers/coverage.rb:94:in `proc': tried to create Proc object without a block (ArgumentError)
```

I tried out #385 but am still seeing this issue with it.

**Comment by nunosilva800** [comment]
Thanks, this is useful in preparing a release compatible with ruby 3

**Comment by jasonlor** [comment]
@etagwerker issue looks like it has been resolved on your latest commit!

**Comment by etagwerker** [comment]
@jasonlor Good to know. I just released v4.6.1: https://github.com/whitesmith/rubycritic/blob/main/CHANGELOG.md#v461--2021-01-28-commits 

This issue should be fixed in that release too. 

Thanks!
